Tools/nib2cib/NSBox.j

```diff
@@ -71,9 +71,8 @@
         }
 
         _title         = [[aCoder decodeObjectForKey:@"NSTitleCell"] objectValue] || @"";
-        _titlePosition = [aCoder decodeObjectForKey:@"NSTitlePosition"];
 
-        if (_titlePosition === undefined)
+        if (_titlePosition === undefined && _boxType !== CPBoxSeparator)
```

Comment:
I used mrcarlberg's suggestion from [this thread](https://groups.google.com/forum/#!topic/objectivej/_1FNQDcD7_4)
I haven't reviewed the full implementation of CPBox and was hesitant about using this code without a full understanding.

The problem with the current implementation is that when CPBox is used, it should have no title, so there should be no titlePosition. What would there be to decode?

I am still somewhat concerned there might be implications I haven't fully grokked.